app/models/katello/glue/pulp/rpm.rb

```diff
@@ -0,0 +1,38 @@
+module Katello
+  module Glue::Pulp::Rpm
+    extend ActiveSupport::Concern
+    include LazyAccessor
+
+    included do
+      lazy_accessor :pulp_facts, :initializer => :backend_data
+
+      lazy_accessor :description, :license, :buildhost, :vendor, :relativepath, :children, :checksumtype,
+                    :changelog, :group, :size, :url, :build_time, :group,
+                    :initializer => :pulp_facts
+    end
+
+    def requires
+      if pulp_facts['requires']
+        pulp_facts['requires'].map { |entry| Katello::Util::Package.format_requires(entry) }
+      else
+        []
+      end
```

Comment:
We generally try to avoid things like:
```
def foo
  if a
    return 'foo'
  end
  blah.each do |item|
     do_something else
   end
   return 3 if needed
   do more stuff
   return 4
end
```

and tend to prefer a single return statement.